src/cli/commands/install.js

```diff
@@ -742,7 +742,7 @@ export class Install {
     if (this.flags.har) {
       steps.push(async (curr: number, total: number) => {
         const formattedDate = new Date().toISOString().replace(/:/g, '-');
-        const filename = `yarn-install_${formattedDate}.har`;
+        const filename = path.join(this.config.cwd, `yarn-install_${formattedDate}.har`);
```

Comment:
This change was done for the test to pass, otherwise the file was generated where the test was executed from, and not the `cwd` that was passed on from the test setup.
I believe this is a safe change, let me know otherwise.